freqtrade/persistence/trade_model.py

```diff
@@ -589,11 +619,11 @@ def to_json(self, minified: bool = False) -> Dict[str, Any]:
             'is_short': self.is_short,
             'trading_mode': self.trading_mode,
             'funding_fees': self.funding_fees,
-            'open_order_id': self.open_order_id,
             'amount_precision': self.amount_precision,
             'price_precision': self.price_precision,
             'precision_mode': self.precision_mode,
-            'orders': orders,
+            'orders': orders_json,
+            'open_orders': open_orders_json
```

Comment:
Maybe stupid question - but why exactly do we need `open_orders` ? 

`orders` already includes open orders - so it's just a question of filtering this.
Adding both however means that the same open order object will be serialized twice - for every RPC call - which is every few seconds if freqUI is open ...

Reply:
Yes you are right it is duplicated data, open orders could be retrieved by filtering from 'orders'